content/citation-tags.tsv

```diff
@@ -2,6 +2,7 @@ tag	citation
 Abe	doi:10.1101/gr.634603
 Abramoff2016_dr	doi:10.1167/iovs.16-19964
 Agarwal2015_targetscan	doi:10.7554/eLife.05005
+Aitchison2017 @url:http://papers.nips.cc/paper/6940-model-based-bayesian-inference-of-neural-activity-and-connectivity-from-all-optical-interrogation-of-a-neural-circuit
```

Comment:
Try switching this line from space-separated to tab-separated.  That should fix the [build error](https://travis-ci.org/greenelab/deep-review/builds/328207073#L711).